tests/cmd/test_augment.py

```diff
@@ -636,6 +638,13 @@ def test_cdx_augment_metadata_tools_components_empty_metadata() -> None:
     assert len(metadata["tools"]["components"]) == 1
 
 
+def test_cdx_augment_metadata_properties_release_id() -> None:
+    sbom: dict[str, Any] = {}
+    CycloneDXVersion1()._augment_metadata_properties_release_id(sbom, "release-id-1")
+    assert sbom["metadata"]["properties"][0]["name"] == "release_id"
+    assert sbom["metadata"]["properties"][0]["value"] == "release-id-1"
```

Comment:
I'd go with a bit of a looser check that still correct:
```suggestion
    assert {
        "name": "release_id",
        "value": "release-id-1"
    } in sbom["metadata"]["properties"]
```